docs/scripts/arkonians.js

```diff
@@ -12,20 +12,88 @@
   //chest variables
   const chestProof = document.getElementById('chest-proof')
   const chestButton = document.getElementById('chest-button')
+  const apiURL = 'http://139.59.233.80:4000'
 
   //------------------------------------------------------------------//
   // Functions
 
   const connected = async function(newstate) {
     //update state
     Object.assign(state, newstate)
-    //toggle html state
-    document.querySelectorAll('.connected').forEach(
-      el => (el.style.display = 'block')
-    )
-    document.querySelectorAll('.disconnected').forEach(
-      el => (el.style.display = 'none')
-    )
+
+    // get user details
+    var detailsRequest = new XMLHttpRequest()
```

Comment:
Why not fetch?

Reply:
hmm, I did try fetch. but when I did POST request kasi API is having issues reading form data.
but has no issues with XMLHttpRequest.

although  the current PR is for get method, it'll be inconsistent if for get request we use fetch and then XMLHttpRequest for POST.